main.go

```diff
@@ -25,6 +26,7 @@ var (
 	setName  = flag.String("r", "", "Amazon SES Configuration Set Name")
 	startTLS = flag.Bool("s", false, "Require TLS via STARTTLS extension")
 	onlyTLS  = flag.Bool("t", false, "Listen for incoming TLS connections only")
+	pinpoint = flag.Bool("p", false, "Use AWS Pinpoint instead of SES")
```

Comment:
I'm thinking that instead of a boolean flag, it might be better to use a String parameter with the two possible values `ses` and `pinpoint` (and `ses` as the default).

The reason this might be better is that using pinpoint is only a boolean choice until we find the need to add a third relay service, which would then make it awkward to add another exclusive command-line argument.

I'm not sure which character to use though.
The most fitting one would be `r` (for relay), or `a` for API, or `i` for Interface, but all three are in use already.
I would be ok with changing the argument for `Amazon SES Configuration Set Name` to `e` to make `r` free.

So my change suggestion would be the following:

```go
var (
        // ...
	relay = flag.String("r", "ses", "Relay API to use (ses|pinpoint)")
	setName  = flag.String("e", "", "Amazon SES Configuration Set Name")
        // ...
)
```

Reply:
Have changed this and the reference to use a switch statement - it's cleaner and allows for easier extend-ability. I assumed this may be an issue, I'm happy with the implementation now 🙂